content/docs/for-platform-operators/tutorials/lifecycle/_index.md

```diff
@@ -5,4 +5,177 @@ expand=true
 include_summaries=true
 +++
 
-Coming soon! In the meantime check out this excellent [blog post](https://medium.com/buildpacks/unpacking-cloud-native-buildpacks-ff51b5a767bf) from our community.
+A `platform` orchestrates builds by invoking the [lifecycle][lifecycle] binary together with buildpacks and application source code to produce a runnable `OCI image`.
+
+<!--more-->
+
+The majority of Buildpack users use platforms, such as [pack][pack] and [kpack][kpack], to run Buildpacks and create `OCI images`. However this might not be desireable especially for users maintaining their own platforms and seeking more control over how the underlying Buildpack `lifecycle phases` are executed.
+
+In this step-by-step tutorial, you will build a `Bash` application without using any `platform` tooling like `pack` or `kpack`. You will also leverage the individual `lifecycle phases` to produce a runnable application image.
```

Comment:
Can we explicitly state up-front that this guide derives from https://medium.com/buildpacks/unpacking-cloud-native-buildpacks-ff51b5a767bf?

Reply:
Sure. I had this note initially at the top; then removed it to the bottom because I didn't want to distract the reader. Added at the start.